addon/templates/components/frost-expand.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="frost-expand-label"  data-test={{hook (concat hook '-label')}} onClick={{action '_onClick'}}>
+  {{frost-icon
+    class='frost-expand-label-icon'
```

Comment:
It's an SVG so resize is par for the course - we should probably just use chevron here so that we don't end up with a bunch of dupes of the same icon